vms/proposervm/block_test.go

```diff
@@ -101,7 +102,8 @@ func TestPostForkCommonComponents_buildChild(t *testing.T) {
 		context.Background(),
 		parentID,
 		parentTimestamp,
-		pChainHeight-1,
+		pChainHeight,
```

Comment:
We changed `pChainHeight-1` to `pChainHeight` here... was that expected? I don't know why this would be impacted by this PR... But maybe this is a bug fix?

Reply:
Talked offline. This is needed because the above mock call sets the expected block context, and with epoching enabled we will pass in the parent pchain height.